### ISSUE TITLE
Nextcloud-client integration support

### ISSUE THREAD

**Issue by aleprovencio** [flathub/org.kde.dolphin#143]
Hi!
Is it even possible to have both Dolphin and Nextcloud-client installed via flatpaks have this kind of integration?
I'm talking about this:
![large-5fa6f3ba1ca1eb7f5f767e198e2583c7](https://user-images.githubusercontent.com/1352683/222900115-55e95dbb-e78c-4104-b3a4-e64b0eb69d11.png)

**Comment by travier** [comment]
We would need to have the nextcloud client included in the Dolphin Flatpak which is unlikely to ever happen.

**Comment by RokeJulianLockhart** [comment]
@travier, why? That would waste space for those who don't use it.

**Comment by Justinzobel** [comment]
It would indeed require more space. Unless someone can find a way to have the Nextcloud and Dolphin Flatpaks talk over D-Bus to get information like sync status, this would be the only way.

**Comment by RokeJulianLockhart** [comment]
Yeah, I think that's the sole viable method unless we want this to baloon.
